packages/block-library/src/image/index.php

```diff
@@ -124,6 +124,7 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 
 	$aria_label = __( 'Enlarge image' );
 
+	$processor->next_tag( 'img' );
```

Comment:
At line 138 the $processor instance is used again but now it points to the img. To my understanding, a new instance should be created before using `get_updated_html()`?